Formula/libfabric.rb

```diff
@@ -17,6 +17,12 @@ class Libfabric < Formula
   depends_on "automake" => :build
   depends_on "libtool"  => :build
 
+  # https://github.com/ofiwg/libfabric/pull/6109 is merged upstream, remove this with next release
+  patch do
+    url "https://github.com/ofiwg/libfabric/pull/6109.diff?full_index=1"
```

Comment:
```suggestion
    url "https://github.com/ofiwg/libfabric/commit/85c9732fd95f9970f5bcf793ca580d45ed7418f2.diff?full_index=1"
```

Also:

- Should this be in a separate PR?
- Does this need a revision bump?

Reply:
> Does this need a revision bump?

Yes, this patch is one that would warrant a revision bump.